samples/ipc/ipc_service/src/main.c

```diff
@@ -6,20 +6,25 @@
 
 #include <zephyr/kernel.h>
 #include <zephyr/device.h>
+#include <string.h>
+
+#include <zephyr/logging/log.h>
 
 #include <zephyr/ipc/ipc_service.h>
 
-#include <zephyr/logging/log.h>
-LOG_MODULE_REGISTER(host, LOG_LEVEL_INF);
+#define STACKSIZE	(4096 + CONFIG_TEST_EXTRA_STACK_SIZE)
```

Comment:
Maybe we can have smaller stack here? If we would like to run samples on VPRs then we can have some issues with memory. Maybe  1024 + CONFIG_TEST_EXTRA_STACK_SIZE would be enough?

Reply:
done